vertical-pod-autoscaler/pkg/recommender/main.go

```diff
@@ -37,13 +37,17 @@ var (
 	prometheusAddress      = flag.String("prometheus-address", "", `Where to reach for Prometheus metrics`)
 	storage                = flag.String("storage", "", `Specifies storage mode. Supported values: prometheus, checkpoint (default)`)
 	address                = flag.String("address", ":8942", "The address to expose Prometheus metrics.")
+	kubeApiQps             = flag.Float64("kube-api-qps", 5.0, `QPS limit when making requests to Kubernetes apiserver`)
+	kubeApiBurst           = flag.Float64("kube-api-burst", 10.0, `QPS burst limit when making requests to Kubernetes apiserver`)
 )
 
 func main() {
 	kube_flag.InitFlags()
 	glog.V(1).Infof("Vertical Pod Autoscaler %s Recommender", common.VerticalPodAutoscalerVersion)
 
 	config := createKubeConfig()
+	config.QPS = float32(*kubeApiQps)
```

Comment:
Can you do the assignment inside of createKubeConfig function and pass kubeApiQps and kubeApiBurst through parameters?

Reply:
Yep, I will take a look today.